packages/lab/src/static-list/StaticListItemContent.css

```diff
@@ -0,0 +1,8 @@
+.saltStaticListItemContent {
+  flex: 1 0;
+  margin: var(--salt-spacing-75) 0;
+  font-size: var(--salt-text-fontSize);
```

Comment:
thinking about most typography is set here but not color. in the changeset, `Text` is used to set certain type stuff? wondering whether we thought about which text CSS should be set here vs not

Reply:
I left color out so icon could also inherit it, but will double check with @ivan-calderon

Reply:
Do you expect icon to be in content? Will the styling be correct?

Also do we need letter spacing here

Reply:
In our best practice, we suggest adding the icons before labels, we opted for adding the typography defaults that `Text` uses here as a fall back to anyone not using text to wrap their labels.

Reply:
letter spacing is needed then

Reply:
is there a reason `Text.css` doesn't have it except when action? should this be up in the main styles?